bin/expedius-install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+##----------------------------------------------------------------------
+## Expedius Install script
+##
+## Author: Dennis Warren @ Colcamex Resources
+## Date: February 2012
+## Updated: January 2020
+## This script is used to install Expedius Lab Auto Downloader onto
+## a server environment that supports Oscar EMR version 15*
+##----------------------------------------------------------------------
+
+# init variables
+TOMCAT_PATH=\/usr\/local\/tomcat\/webapps
+ADMIN_EMAIL=none
+APP_DATA=\/var\/lib\/expedius
+EMAIL_SERVICE=no
+EXPEDIUS_USERNAME=admin
+EXPEDIUS_PASSWORD=3xp3dius#1
```

Comment:
Is this a real secret? Or doesn't matter?

Reply:
Yes it is an issue. I left it hard coded until we can discuss how interaction with the installer works.  I'm thinking command prompts during install with a set-up file executed at the end of the script.